locations/spiders/gls-deu.py

```diff
@@ -0,0 +1,66 @@
+import scrapy
+import json
+import csv
+
+class GenspiderSpider(scrapy.Spider):
+    name = 'glsdeu'
+    allowed_domains = ['api.gls-pakete.de']
+    index = 1
+    start_urls = ['https://api.gls-pakete.de/parcelshops?version=4&coordinates=50.88246,8.91596&distance=50']
+    usedLongLat = []
+    deList = []
+
+    with open('../searchable_points/eu_centroids_40km_radius_country.csv', 'r') as openFile:
```

Comment:
This should be in a `start_requests` function, see [this example](https://github.com/alltheplaces/alltheplaces/blob/fdf689dd0f01102349a47851bf012cd59e5246cb/locations/spiders/starbucks.py#L16).